scripts/fit.py

```diff
@@ -332,14 +339,21 @@ def get_model_str(N_PARS, pars):
             )
         return model
 
+    # interpolate the fit to 100 points for better plots
+    x_line = np.logspace(np.log10(xVal.min()), np.log10(xVal.max()), 100)
+    if xrange:
+        x_line = np.logspace(np.log10(xrange[0]), np.log10(xrange[1]), 100)
```

Comment:
I would maybe not create the points in a log-space. In case some other observables is generated, a logspace might not make sense and the samples would be weirdly sparse in most regions. I would just keep it linear and when plotting it with a logarithmix axis the sampling would be in the worst linearly sparse as in the binning of the log-axis.

Reply:
All plots are hardcoded in log space anyway. Feel free to rewrite this and all the plotting scripts to make this configurable. This is definitely out of scope for this PR.

Reply:
Resolving this comment, as I definitely won't make those changes in the scope of this PR.